components/formInput.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <b-form-group :id="id + 'Group'">
```

Comment:
You can use string interpolation . For example : 
Create a computed property which will return: 
`${id}Group`